src/js/stores/CosmosPackagesStore.js

```diff
@@ -355,6 +372,18 @@ class CosmosPackagesStore extends GetSetBaseStore {
     this.emit(COSMOS_REPOSITORIES_SUCCESS);
   }
 
+  processServiceDescriptionSuccess(cosmosPackage, name) {
```

Comment:
Why are we using this method to do the same thing as `processPackageDescriptionSuccess`?

Reply:
The only different is the emitted event. I think they should be kept using their own separate methods.

Reply:
Talked offline and decided we will introduce a new field `serviceDetails` to make this nicer.